docs/articles/new-expensify/expensify-card/Change-Expensify-Card-limit.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Change Expensify Card limit
+description: Increase or decrease the limit for an Expensify Card or for a group
+---
+<div id="new-expensify" markdown="1">
```

Comment:
These steps are in Classic, let's update the route and div id

Reply:
Updated!